source/connect.txt

```diff
@@ -49,68 +49,69 @@ Connect
 
 When you open |compass-short|, an initial connection dialog appears:
 
-.. figure:: /images/compass/paste-connection-string.png
+.. figure:: /images/compass/new-connection.png
    :figwidth: 690px
-   :alt: Connection dialog for pasting your connection string
+   :alt: New connection dialog for pasting your connection string
 
-|compass-short| provides two methods to connect to your deployment,
-either providing your deployment connection string or filling in your
-deployment information in specific fields.
+|compass-short| provides multiple methods to connect to your deployment,
+either providing your deployment connection string or through Advanced 
+Connection Options. 
```

Comment:
I suggest we modify this to make the user the actor in the sentence, rather than Compass. We can try something like:

```
You can connect to your deployment by either:

- Providing your deployment connection string
- Specifying :guilabel:`Advanced Connection Options`
```